linera-execution/src/wasm/sanitizer.rs

```diff
@@ -404,7 +404,7 @@ impl<'bytecode> Sanitizer<'bytecode> {
     }
 }
 
-#[cfg(all(test, with_wasmer))]
+#[cfg(test)]
```

Comment:
We always test with Wasmer?

Reply:
Now we have to :/ Because otherwise we can't enable the `wat` feature only for tests :(

Reply:
:(

Relevant discussion: https://github.com/rust-lang/cargo/issues/1596